.github/workflows/publish-to-npm.yml

```diff
@@ -18,3 +18,4 @@ jobs:
               with:
                   token: ${{ secrets.ADOBE_BOT_NPM_TOKEN }}
                   access: 'public'
+                  tag: 'alpha'
```

Comment:
We need to find a better way to handle this 😅

Reply:
Yeah I know 😢

Reply:
No probably a different ticket.